src/building/compiler-documenting.md

```diff
@@ -1,16 +1,16 @@
 # Documenting rustc
 
 You might want to build documentation of the various components
-available like the standard library. There’s two ways to go about this.
-You can run rustdoc directly on the file to make sure the HTML is
+available, like the standard library. There are two ways to go about this.
+You can run `rustdoc` directly on the file to make sure the HTML is
 correct, which is fast. Alternatively, you can build the documentation
-as part of the build process through x.py. Both are viable methods
+as part of the build process through `x.py`. Both are viable methods
 since documentation is more about the content.
 
 ## Document everything
 
-This uses the beta rustdoc, which usually but not always has the same output
-as stage 1 rustdoc.
+This uses the beta `rustdoc`, which usually, but not always, has the same output
+as stage1 `rustdoc`.
```

Comment:
I don't think stage 1 -> stage1 makes it any more or less clear. I would prefer to keep "stage 1" because `stage1` is normally the name of a rustup toolchain, I wouldn't use it in prose.